classes/music_album.rb

```diff
@@ -0,0 +1,29 @@
+require './item'
+
+class Album < Item
+  attr_reader :on_spotify
```

Comment:
[Optional] I believe it would be better if you have a writer for (on_spotify) so you can alter the value later